src/controllers/tripController.js

```diff
@@ -12,10 +14,36 @@ export const oneWayTripRequest = async (req, res) => {
   const { id } = req.auth;
   const payload = { ...req.body, status: 'pending', userId: id };
   try {
-    const tripRequest = await tripServices.postTrip(payload);
+    const tripRequest = await postTrip(payload);
 
     return respondWithSuccess(res, statusCode.created, 'request successfully sent', tripRequest.toJSON());
   } catch (error) {
     return respondWithWarning(res, statusCode.internalServerError, 'Internal Server Error');
   }
 };
+
+export const approveTripRequest = async (req, res) => {
+  const status = 'approved';
+  try {
+    const tripRequest = await updateTripStatus(req.params.tripId, status);
+    const { 1: { dataValues } } = tripRequest;
```

Comment:
how about tripRequest.toJSON()? Also, I'd like to understand why we'd have the `dataValues` nested in a property `1`?

Reply:
I tried using `tripResponse.toJSON()` but it was throwing a `500` error so i decided to leave it this way